src/NServiceBus.Core/v6-obsoletes.cs

```diff
@@ -110,6 +110,12 @@ public abstract class DeliveryOptions
            RemoveInVersion = "7.0",
            TreatAsErrorFromVersion = "6.0")]
         public string ReplyToAddress { get { throw new NotImplementedException(); } set { throw new NotImplementedException(); } }
+
+        [ObsoleteEx(
+           Message = "Turn best practices check off using configuration.DisableFeature<BestPracticesEnforcement>()",
```

Comment:
`configuration.DisableFeature<BestPracticesEnforcement>()` not very discoverable IMO, should we add an extension to busconfiguration instead ?